### DISCUSSION TITLE
VX3.5.0.44无法安装防撤回

### DISCUSSION THREAD

**Discussion by matudou-arch** [huiyadanli/RevokeMsgPatcher#291] [Q&A]
微信打开都可以点
![微信图片_20220209185325](https://user-images.githubusercontent.com/68991789/153184284-92bc2065-4bb6-464d-96d9-40198ae4a0cf.png)
![微信图片_20220209185421](https://user-images.githubusercontent.com/68991789/153184326-9a82e788-2001-4eef-a306-afb840edae81.png)
![微信图片_20220209185424](https://user-images.githubusercontent.com/68991789/153184354-66a32bbf-448b-424f-b9e1-f88d9613bdc0.png)

**Comment by xuwenkai357** [comment]
收到了

**Reply by matudou-arch** [reply]
谢谢大佬

**Comment by matudou-arch** [comment]
![微信图片_20220209224155](https://user-images.githubusercontent.com/68991789/153224130-45824204-2048-4b1a-9c99-c5aedf41e7a8.png)
我再提供一下微信版本，话说这个软件识别是44，但是版本是46，会不会是这个问题呢

**Reply by huiyadanli** [reply]
> ![微信图片_20220209224155](https://user-images.githubusercontent.com/68991789/153224130-45824204-2048-4b1a-9c99-c5aedf41e7a8.png) 我再提供一下微信版本，话说这个软件识别是44，但是版本是46，会不会是这个问题呢

参考这里的解决方案，是一样的问题：https://github.com/huiyadanli/RevokeMsgPatcher/issues/287

**Comment by Kaltemraichu** [comment]
撤回功能可以打修补补丁，但是双开push ebp改ret不行了，做成修补补丁后，无法打开微信，版本是
![image](https://user-images.githubusercontent.com/100063801/154832458-d0765871-431d-4a14-8389-f8427324f504.png)
